src/services/action/base-action.ts

```diff
@@ -23,7 +23,7 @@ export class BaseAction implements Action {
     itemType: string;
     actionType: string;
     view: string;
-    geolocation: Serializable;
+    geolocation: Serializable | null;
```

Comment:
Nitpick: use nullable syntax? 
```suggestion
    geolocation?: Serializable;
```